### ISSUE TITLE
Gnome desktop environment not idetified in Ubuntu 16.04

### ISSUE THREAD

**Issue by dellekappa** [dzsessona/QuickOpener-NetBeans#77]
In ubuntu 16.04 there is no "gnome-session" processes so the class com.sessonad.oscommands.detector.OSDector fails to identify the correct desktop environment.

**Comment by yanosh-k** [comment]
I'm experiencing the same problem.
Adding a check for `gnome-session-binary` in the OSDetector seems to fix the issue.

```java
if (checkProcessNames("gnome-session", "gnome-session-binary"))
```

**Comment by markiewb** [comment]
fixed in 1.1.0.3
